admin_guide/assembly_replace-master-host.adoc

```diff
@@ -0,0 +1,28 @@
+[[assembly_replace-master-host]]
+= Replacing a master host
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: replace-master-host
+
+toc::[]
+
+
+You can replace a failed master host.
+
+First, remove the failed master host from your cluster, and then add a replacement
+master host. If the failed master host ran etcd, scale up etcd by adding etcd
+to the new master host.
```

Comment:
From this, I think yuo're saying that all steps in this topic need to be performed? That might be a little hard to figure out unless they read the intro here, which I would not guarantee people would. This might be presented like this because of the modularization. Can I suggest it be clearer in the modules that there might be "next steps" which would be the next parts of the section? That could be better positioned in the assembly, to keep the "modularization" of the sections, especially if the modules are not overly specific to this section.

Hope this makes sense.

Reply:
It does make sense, but I think this is a bigger decision that we need to make about how to handle these 1 story = 1 topic assemblies. I'm sending you an email so we can talk about this more, and, for now, I'm going to add this note to the top:
```
[IMPORTANT]
====
You must complete all sections of this topic.
====
```